Installation/include/CGAL/config.h

```diff
@@ -362,7 +362,15 @@
 #  elif BOOST_ENDIAN_LITTLE_BYTE
 #    define CGAL_LITTLE_ENDIAN
 #  else
-#    error Unknown endianness
+#    ifdef CGAL_DEFAULT_ENDIAN
+#      if CGAL_DEFAULT_ENDIAN == 0
```

Comment:
This `CGAL_DEFAULT_ENDIAN` should be renamed `CGAL_DEFAULT_IS_BIG_ENDIAN`, or something similar.